tests/common/test_data.py

```diff
@@ -21,7 +21,14 @@
 RANDOM_SWITCH_ON_PROBABILITIES = SwitchOnProbabilities(
     {key: random.random() for key in SWITCH_ON_PROBABILITY_KEYS}
 )
-LOAD_PROFILE_STOVE = TimeSeries({quarterly_hour: 700 for quarterly_hour in range(4)})
+LOAD_PROFILE_STOVE = TimeSeries(
+    [
+        TimeSeriesEntry(timedelta(), 100),

```

Comment:
value of `TimeSeriesEntry` seems to be in kW. Since we're in a household I would suggest - if you don't mind -  to have the test data to a more realistic value :)